test/resources/annotations/package.json

```diff
@@ -2,6 +2,9 @@
   "dependencies": {
     "@cap-js/graphql": "*"
   },
+  "devDependencies": {
+    "@cap-js/sqlite": "*"
```

Comment:
Are these added for the plugin detection ?
If that is the reason it would be better to run the test with `CDS_CONFIG` to allow for testing with not just `@cap-js/sqlite`.

Reply:
I don't want to introduce those internal flags also for open source modules.
In the meanwhile, I'd hope that there's no need anymore to switch back to the old implementation :)